<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="dialogBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dialog</string>
   </property>
  </widget>
  <widget class="QPushButton" name="successBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>success</string>
   </property>
  </widget>
  <widget class="QPushButton" name="informationBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="errorBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>error</string>
   </property>
  </widget>
  <widget class="QPushButton" name="warningBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>warning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aeroBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AeroWindow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="winWindowBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>WinWindow</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
